cli/command/node/health.go

```diff
@@ -78,3 +82,52 @@ func runHealth(storageosCli *command.StorageOSCli, opt *healthOptions) error {
 	}
 	return formatter.NodeHealthWrite(nodeHealthCtx, node)
 }
+
+// UpdateNodeHealth updates the health status of a given node by querying the
+// node endpoints.
+func UpdateNodeHealth(node *cliTypes.Node, address string, timeout int) error {
+	const healthEndpointFormat = "http://%s:%s/v1/health"
```

Comment:
Maybe construct the url using the constant here? https://github.com/storageos/go-api/blob/5e025fb3fc6188d3af07b19a15e752370a5c9019/health.go#L14

Reply:
Thanks!